FirebaseCore/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+- In preparation for supporting Privacy Manifests, remove `Info.plist` files
```

Comment:
Clarify that this is for binary distributions.

Reply:
Done.